### PR TITLE
Use rollup to build cjs&es modules

### LINKED ISSUE #13: Provide es module built
es module built make tree-shaking available in build tools like webpack/rollup

### PR DESCRIPTION
**Pull request by foisonocean**
fix #13

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const listOfEnties = readdirSync(resolve(__dirname, './build'));
+
+export default listOfEnties.map(entry => {
+  let output;
+  switch (entry) {
+    case 'index.js':
+      output = [
+        {
+          file: resolve(__dirname, './publish/lib/index.cjs.js'),
+          format: 'cjs',
```
Comment (levrik) [inline rollup.config.js]: I though tree shaking works only with ES modules?

File: rollup.config.js [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+const listOfEnties = readdirSync(resolve(__dirname, './build'));
+
+export default listOfEnties.map(entry => {
+  let output;
+  switch (entry) {
+    case 'index.js':
+      output = [
+        {
+          file: resolve(__dirname, './publish/dist/index.cjs.js'),
+          format: 'cjs',
```
Comment (levrik) [inline rollup.config.js]: I though tree shaking works only with ES modules?

File: rollup.config.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+const listOfEnties = readdirSync(resolve(__dirname, './build'));
+
+export default listOfEnties.map(entry => {
+  let output;
+  switch (entry) {
+    case 'index.js':
+      output = [
+        {
+          file: resolve(__dirname, './publish/dist/index.cjs.js'),
+          format: 'cjs'
```
Comment (levrik) [inline rollup.config.js]: I though tree shaking works only with ES modules?

Comment (foisonocean) [reply rollup.config.js]: I think the we still need to provide a entry file for who's build tools doesn't support es module syntax(e.g. webpack1, jest or use in Node.js directly), and then people can still use the same import useage (`import { A, B } from 'mdi-react'`) which is more convenient than `import A from 'mdi-react/A';import B from 'mdi-react/B'`. And people can also use [babel-plugin-import](https://www.npmjs.com/package/babel-plugin-import) make tree-shaking available even with commonjs built.

**Comment by levrik** [comment]
Thank you

**Comment by levrik** [comment]
I just published a new version under `@levrik/mdi-react` with the version number `3.0.0-build.1`.
Could you try if it's working for you? I will publish it under the normal package if everything works fine 🙂

**Comment by foisonocean** [comment]
@levrik It works fine for me😄

**Pull request merged**